include/ebpf_program_attach_type_guids.h

```diff
@@ -31,29 +31,41 @@ extern "C"
     __declspec(selectany) ebpf_attach_type_t EBPF_ATTACH_TYPE_BIND = {
         0xb9707e04, 0x8127, 0x4c72, {0x83, 0x3e, 0x05, 0xb1, 0xfb, 0x43, 0x94, 0x96}};
 
-    /** @brief The programs attached to the INETx_CONNECT hook will be invoked for
-     * connect() calls on TCP or UDP sockets or when a UDP socket calls sendto() to
+    /** @brief The programs attached to the INET4_CONNECT hook will be invoked for
+     * connect() calls on TCP or UDP sockets or when a UDP socket sends a packet to
      * a unique remote address/port tuple. May be scoped to a network compartment. The programs return 1 to permit a
      * connection, and 0 to block it.
      */
     __declspec(selectany) ebpf_attach_type_t EBPF_ATTACH_TYPE_CGROUP_INET4_CONNECT = {
         0xa82e37b1, 0xaee7, 0x11ec, {0x9a, 0x30, 0x18, 0x60, 0x24, 0x89, 0xbe, 0xee}};
 
+    /** @brief The programs attached to the INET6_CONNECT hook will be invoked for
```

Comment:
Simply `\ref` to the documentations of attach types? Same applies to program type GUIDs.

Reply:
Depends on whether we think every GUID will have an integer value too.   If we don't require integer values for some future Windows-only GUID then it may be better to keep documentation here too.